app/views/curation_concerns/base/_form.html.erb

```diff
@@ -34,7 +34,7 @@
       </div>
 
       <div id="savewidget" class="col-xs-12 col-sm-4 fixedsticky" role="complementary">
-        <%= render 'form_progress', f: f %>
+        <%= render 'form_progress', f: f, agreement_accepted: @form.agreement_accepted %>
```

Comment:
`#agreement_accepted` is not a model property, so I'm surprised this isn't throwing an error. If it were a model property you wouldn't need to pass it as a local. You could access it via `f.object.agreement_accepted`